xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.css

```diff
@@ -51,10 +51,11 @@
   width: 15% !important;
 }
 
-/* This button has no border or background, but wraps an image to allow keyboard navigation. */
-#usersandgroupstable button.rights-edit {
-  border-color: transparent;
-  background: transparent;
+#usersandgroupstable button.rights-edit.yes {
+  color: $theme.notificationSuccessColor;
```

Comment:
Feels a bit weird here to use `notification(Success|Error)Color`, now maybe that's the best practice in themes?

Reply:
That's some of the old colortheme variables. I'm not sure we have a best practice for this. I think I did use those old variables since others are used all throughout the file and I'd rather keep consistent inside one file.

To be honest this whole file looks quite old and breaks current CSS codestyles in multiple places.
I opened a topic on the forum about introducing a new [type of label for codestyle](https://forum.xwiki.org/t/jira-label-proposal-codestyle/14167), this file could really use a ticket :)

I just checked our [HTML and CSS best practices](https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/) and the entry `don't hard code colors in properties - use [ColorTheme variables](https://extensions.xwiki.org/xwiki/bin/view/Extension/Color%20Theme%20Application#HUsingColorThemesvariables)` points to a doc that uses the old colortheme variables. We might want to update the link or the doc.

I'll look a bit more into the state of deprecation of this old colortheme, it's not clear to me what it is yet. Using the old colortheme everywhere does not look like something we should recommend...

Reply:
I updated the doc to include a link to how to use the new variables.